src/responses.ts

```diff
@@ -1,3 +1,6 @@
+import { Block } from '@ethersproject/abstract-provider'
+import { Chunk } from 'near-api-js/lib/providers/provider'
```

Comment:
is this import correct? I think it should just be `import { Chunk } from 'near-api-js'`

Reply:
Yes, it is. If we tried to import from near-api-js, we could get the following error:
`Module '"near-api-js"' has no exported member 'Chunk'.`